index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="toolbar">
+      <a href="#">
+      <img src="./images/my-logo.svg" alt="my logo" class="my-logo">
+      </a>
+      <nav>
+         <a href="#">
+            <img 
+            src="./images/hamburger.svg" 
+            alt="hamburger menu"
+            class="hamburger-img" id="hamburger">
+            </a>
+      </nav>
+      </header>
+         <section class="main">
+            <header>
+               <h1 class="headline">I'm Ezinwa charles <br>Glad to see you!</h1>
+               <p class="desc">
+                  I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+               </p>
+               <p class="connect-msg">LET'S CONNECT</p>
+            </header>
```

Comment:
You have done a great job. Please remove the header tag from your section since you already have done the header on the above and this section is not the header. you can use `div` tag instead of it. Kindly fix it